scripts/common.py

```diff
@@ -418,6 +418,13 @@ def check_prs_for_current_release_are_released(repo):
                 'released before release summary generation.')
 
 
+def convert_filepath_to_url(file_path):
```

Comment:
Same thing -- update names etc.

This function should also have unit tests and should be robust. Is it possible for filenames in Unix to contain backslashes and, if so, will this function do the wrong thing?

Reply:
I will add an os check to run this only on Window. But I have no idea how to rename this function now, maybe you can provide some suggestions?

Reply:
Sure thing! I've added notes in the other comments.